guillotina/contrib/catalog/pg.py

```diff
@@ -581,15 +583,28 @@ def load_meatdata(self, query: ParsedQueryInfo, data: typing.Dict[str, typing.An
         results = []
         try:
             context_url = get_object_url(container)
+            request = get_current_request()
+            fullobjects = request.query.get('_fullobjects', 'false').lower() != 'false'
+            txn = get_current_transaction()
         except RequestNotFound:
             context_url = get_content_path(container)
+            request = None
+            fullobjects = False
+            txn = None
         logger.debug(f'Running search:\n{sql}\n{arguments}')
         for record in await conn.fetch(sql, *arguments):
             data = json.loads(record['json'])
-            result = self.load_meatdata(query, data)
-            result['@name'] = record['id']
-            result['@uid'] = record['zoid']
-            result['@id'] = data['@absolute_url'] = context_url + data['path']
+            if fullobjects and request is not None and txn is not None:
+                # Get Object
+                obj = await txn.get(data['uuid'])
+                # Serialize object
+                view = DefaultGET(obj, request)
```

Comment:
Maybe we should just call serializer on this? I'm not a fan of using view code as it could have side affects.

Reply:
I was not willing to update the logic of include and omit

Reply:
ahh, so that include/omit can then also be used with these results